platform/linux-generic/m4/odp_mdev.m4

```diff
@@ -0,0 +1,34 @@
+##########################################################################
+# Enable mdev support
+##########################################################################
+mdev_support=no
```

Comment:
Same comments as for PR #380, moreover we shouldn't duplicate commits in two PRs as that will cause merge conflicts. I suggest you reorg these into: 

1. A basic mdev PR that adds the common code. This can be reviewed and merged first.

2. Once that's in, individual PRs for the various mdev drivers become a lot simpler to add on top of this.